### PR TITLE
README.md: added-command-line

### PR DESCRIPTION
**Pull request by Stanislaw-bnk**

### CODE REVIEW

File: file [hunk 1]
```diff
@@ -0,0 +1,1425 @@
+Active Internet connections (w/o servers)
```
Comment (pietrushnic) [inline file:1]: @Stanislaw-bnk this definitely should not be here.

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: file (hunk 1 above)
Comment (Stanislaw-bnk) [reply file]: Don't know where it came from. I have deleted this file.

**Comment by pietrushnic** [comment]
@Stanislaw-bnk changes will not be reviewed if all pre-commit check didn't passed.

**Comment by Stanislaw-bnk** [comment]
@pietrushnic Przypadkiem zauważyłem, że zmiany, które chciałem [tu](https://github.com/Dasharo/docs/pull/515/commits/bc8195e80add21cb354d9e6b07e8bd1a6623a716) wprowadzić dot. błędu `OSError: [Errno 98] Address already in use` są już ujęte w https://3mdeb.gitlab.io/human-resources/processes/#troubleshooting. Także nie ma sensu, aby to powtarzać. Do usunięcia.

**Comment by pietrushnic** [comment]
@Stanislaw-bnk please respect our quality process and procedures:

* This is a public repository in which we use English
* You link to the internal wiki, which does not make sense to anyone who visits this issue
* If readme of this repo miss error handling it should be added

**Comment by Stanislaw-bnk** [comment]
@pietrushnic My bad, I am still learning the git, will double-check the procedures next time. 
I've added the error handling info. This time I've run the `pre-commit hooks`, and all checks have passed.
Hope it will be fine now: https://github.com/Dasharo/docs/pull/515/commits/7f378df3f16b756a0879e67f62ed0511227decf0

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**